app/views/accounts/_header.html.erb

```diff
@@ -16,6 +16,6 @@
     <%= render 'accounts/badges' %>
   </div>
 </div>
-<h1 class="padding-bottom-1 bold tablet:display-none">
+<h1 class="padding-bottom-1 text-bold tablet:display-none">
```

Comment:
It existed previously, but I don't think adding bold to a heading makes any difference here.

```suggestion
<h1 class="padding-bottom-1 tablet:display-none">
```